src/main/java/com/cisco/trex/stateless/TRexClient.java

```diff
@@ -467,16 +511,45 @@ private static EthernetPacket buildArpPkt(String srcMac, String srcIp, String ds
             throw new IllegalArgumentException(e);
         }
 
+        Pair<EtherType, Packet.Builder> payload = buildVlan(arpBuilder, vlan);
```

Comment:
[Minor issue, could be ignored]
Not sure, maybe it would better to rename this method since it doesn't build vlan in all cases. Or maybe we can move part of `buildVlan` logic here, like if there are some vlan tags then perform buildVlan, else do nothing.
I mean this line is confusing without `buildVlan` context.

Reply:
fixed